package.json

```diff
@@ -48,7 +49,10 @@
     "new:util": "hygen util new",
     "prepush": "npm run test:ci",
     "static": "serve -s build",
+    "deps:graph": "depcruise --validate --output-type dot -- src | dot -T svg > dependency-graph.svg",
+    "deps:report": "depcruise --validate --output-type err-html --output-to dependency-report.html -- src",
     "test:ci": "CI=true npm-run-all test:lint test:types \"test:unit -- -i\" test:visual:ci test:e2e:ci",
```

Comment:
Maybe we should add `test:deps` to `test:ci`?

Reply:
Ah good idea.

Reply:
I would recommend adding it toward the beginning of these (maybe after lint?) so that it fails fast 👍

Reply:
Yep, I had added it after the types -- cool with that positioning?
4482a8e